### PR TITLE
@brief is removed completely

### PR DESCRIPTION
**Pull request by annando**
`@brief` is now removed completely. If it wasn't already the case, the `@brief` content had been moved to the top as a summary.

### CODE REVIEW

File: src/Object/Image.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function __construct($data, $type = null)
 	}
 
 	/**
-	 * @brief Destructor
+	 * Destructor
 	 * @return void
```
Comment (nupplaphil) [inline src/Object/Image.php]: Plz add a blank line between tags and summary for better readability

File: src/Object/Image.php [hunk 2]
```diff
@@ -671,7 +671,7 @@ public function asString()
 	}
 
 	/**
-	 * @brief supported mimetypes and corresponding file extensions
+	 * supported mimetypes and corresponding file extensions
 	 * @return array
```
Comment (nupplaphil) [inline src/Object/Image.php]: Plz add a blank line between tags and summary for better readability

File: src/Object/Image.php [hunk 3]
```diff
@@ -681,7 +681,7 @@ public static function supportedTypes()
 	}
 
 	/**
-	 * @brief Maps Mime types to Imagick formats
+	 * Maps Mime types to Imagick formats
 	 * @return array With with image formats (mime type as key)
```
Comment (nupplaphil) [inline src/Object/Image.php]: Plz add a blank line between tags and summary for better readability

File: src/Worker/DBClean.php [hunk 4]
```diff
@@ -42,7 +42,7 @@ private static function forkCleanProcess() {
 	}
 
 	/**
-	 * @brief Remove orphaned database entries
+	 * Remove orphaned database entries
 	 * @param integer $stage What should be deleted?
```
Comment (nupplaphil) [inline src/Worker/DBClean.php]: Plz add a blank line between tags and summary for better readability

File: src/Worker/Cron.php [hunk 5]
```diff
@@ -137,7 +137,7 @@ private static function checkdeletedContacts()
 	}
 
 	/**
-	 * @brief Update public contacts
+	 * Update public contacts
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
```
Comment (nupplaphil) [inline src/Worker/Cron.php]: Plz add a blank line between tags and summary for better readability

**Comment by annando** [comment]
All missing new lines should be added, although the frontend doesn't shows this.

**Comment by MrPetovan** [comment]
Props to @nupplaphil for reviewing this. Hero.

**Comment by annando** [comment]
@MrPetovan just see it as some kind of revenge for his massive PRs ... :grin:

**Comment by MrPetovan** [comment]
I mean, the clear winner is the Friendica code base, so please keep firing away at each other.

**Pull request merged**